applications/nrf5340_audio/tools/buildprog/pcft_sign.py

```diff
@@ -39,17 +39,26 @@ def awklike(field_str, filename):
 
 def sign(orig_pcft_hex, build_dir):
     """ A function to combine and sign PCFT"""
+
+    if os.name == 'nt':
+        folder_slash ='\\'
+    else:
+        folder_slash ='/'
+
     # Make sure we input build_dir as absolute path
-    indices = [i for i, c in enumerate(build_dir) if c == '/']
-    final_file_prefix = build_dir[indices[-2]+1:].replace('/', '_')+'_'
+    indices = [i for i, c in enumerate(build_dir) if c == folder_slash]
+    final_file_prefix = build_dir[indices[-2]+1:].replace(folder_slash, '_')+'_'
 
     # RETEIVE setting value from .config
     # "${ZEPHYR_BASE}/../bootloader/mcuboot/root-rsa-2048.pem"
 
     mcuboot_rsa_key = awklike('CONFIG_BOOT_SIGNATURE_KEY_FILE=', build_dir +
                               '/mcuboot/zephyr/.config')
-    if ('/' in mcuboot_rsa_key) or ('\\' in mcuboot_rsa_key):
+    # Use \\ or / should be worked on OS windows
```

Comment:
Change comment to something like this: "'\' is used for Windows, '\\' is used for other OSs like Linux'"

Reply:
Change comment to more clear.